visualc/VS2010/mbedTLS.vcxproj

```diff
@@ -345,9 +353,13 @@
     <ClCompile Include="..\..\library\x509write_csr.c" />
     <ClCompile Include="..\..\library\xtea.c" />
     <ClCompile Include="..\..\tests\src\certs.c" />
+    <ClCompile Include="..\..\tests\src\asn1_helpers.c" />

```

Comment:
This is a generated file. Instead of resolving the conflict manually, run `scripts/generate_visualc_files.pl`.